packages/connect/CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ## [Unreleased]
 
-## 0.7.6 - 2022-06-16
+## 0.7.7 - 2022-06-16
```

Comment:
No, you don't make versions magically disappear
Please say that this updates the Rococo chain spec